db/sql/sql.go

```diff
@@ -528,17 +538,28 @@ func makeStateColumns(s *schema.Schema) (cols []string) {
 	return cols
 }
 
-func makeJoin(s *schema.Schema, tableName string, q sq.SelectBuilder) sq.SelectBuilder {
+func makeJoin(s *schema.Schema, tableName string, q sq.SelectBuilder, aliasRelationPropertyPrefix string, aliasTableName string) sq.SelectBuilder {
```

Comment:
Actually, aliasTableName is important when top level resource has two children with the same resources which has two or more depth relations. Without newAliasTableName, current test passes because child resource of top level resource in my test schema has only one depth relation. In the next patch, I will modify test schema to check if this aliasTableName is needed.